src/algebra/group/basic.lean

```diff
@@ -538,6 +538,9 @@ by rw [sub_eq_neg_add, neg_add_cancel_left]
 lemma add_sub_cancel'_right (a b : G) : a + (b - a) = b :=
 by rw [← add_sub_assoc, add_sub_cancel']
 
+@[simp] lemma sub_add_cancel' (a b : G) : a - (a + b) = -b :=
+by rw [← neg_sub, add_sub_cancel']
```

Comment:
Can you `to_additive` this?

Reply:
I can add multiplicative versions of most lemmas in this file but in another PR. I'm not sure that we have multiplicative versions of both lemmas I use in `rw`.